### ISSUE TITLE
Don't persist unsaved values on Avatar picker dialog

### ISSUE THREAD

**Issue by erdemyerebasmaz** [breez/breezmobile#1206]
When Avatar name & icon are randomized but cancelled on Avatar picker dialog, their values persist the next time Avatar dialog is shown.

For example, if the user goes through this flow and then decides to upload a new avatar image. They are very likely to end up with the randomized user name on the first flow they cancelled.

https://github.com/breez/breezmobile/assets/4012752/c2ffade7-8e80-4460-b4c1-2cf319ea4a28

**Comment by bhav-khurana** [comment]
@erdemyerebasmaz Can I work on this issue?

**Comment by erdemyerebasmaz** [comment]
> @erdemyerebasmaz Can I work on this issue?

@bhav-khurana Yes, go ahead, it's a low priority issue that's not actively being worked on. Thanks!
